sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -60,7 +60,7 @@ tcuFloat.FloatDescription = function(exponentBits, mantissaBits, exponentBias, f
 tcuFloat.FloatDescription.prototype.zero = function(sign) {
     return tcuFloat.newDeFloatFromParameters(
         deMath.shiftLeft((sign > 0 ? 0 : 1), (this.ExponentBits + this.MantissaBits)),
-        new tcuFloat.FloatDescription(this.ExponentBits, this.MantissaBits, this.ExponentBias, this.Flags)
+        this
```

Comment:
Were all of these methods just allocating unnecessary FloatDescription objects all along? Or is there a subtle semantic change here?